setup.py

```diff
@@ -31,7 +31,7 @@
     'scipy': ['scipy >= 1.4.0'],
     'astropy': ['astropy >= 3.0'],
     'python-casacore': ['python-casacore >= 3.3.1'],
-    'ducc0': ['ducc0 >= 0.8.0'],
+    'ducc0': ['ducc0 >= 0.6.0'],
```

Comment:
Revert reversion of ducc0 from 0.8.0 to 0.6.0

Reply:
I think the double-negative was  confusing here:

```suggestion
    'ducc0': ['ducc0 >= 0.8.0'],
```